### ISSUE TITLE
The behavior of training stage and inference stage

### ISSUE THREAD

**Issue by King4819** [YifanXu74/Evo-ViT#9]
Hi, I want to ask that the behavior of training stage and inference stage. Since the paper illustrate the behavior of training stage, however, the inference stage did not mentioned. Is the inference stage still perform token selection, slow-fast updating and global class attention evolution? Thanks!

**Comment by YifanXu74** [comment]
Yes, the  inference stage still perform the whole process including token selection, slow-fast updating and global class attention evolution.

**Comment by King4819** [comment]
OK, thanks for your reply
